Meta/Lagom/CMakeLists.txt

```diff
@@ -259,6 +259,11 @@ function(lagom_test source)
     )
 endfunction()
 
+function(serenity_test test_src sub_dir)
+    cmake_parse_arguments(PARSE_ARGV 2 SERENITY_TEST "MAIN_ALREADY_DEFINED" "CUSTOM_MAIN" "LIBS")
```

Comment:
Could use a FIXME here to pass along main already defined and custom main when we get to porting tests that use it.